scripts/release-notes.py

```diff
@@ -43,6 +43,20 @@ def num_commits_since_prev_tag(token, base_url):
     print(f"There are {num_commits} new commits since {prev_release_tag}")
     return num_commits
 
+categories = {
```

Comment:
note: this is a dictionary, the order of items is not guaranteed. You may want
```
[
  { title: ..., label: ... }
]
```
And if you make `title` optional it allows you to skip `test` and `skip` entries
```

Reply:
Thanks for the suggestion. Changed it to an array. Also, added a check for the title, if that's empty, it won't be printed to console